README.md

```diff
@@ -1 +1,169 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Startup society summit] <a name="about-project"></a>
+
+**[Startup society summit]** is a static website that represents the website for Startup society summit 2023 event or conference.
+
+## 🛠 Built With <a name="built-with">`HTML & CSS/SCSS` and `JavaScript`</a>
+
+### Tech Stack <a name="tech-stack">`HTML`, `CSS/SCSS`, `JavaScript`</a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3.org/standards/webdesign/htmlcss">HTML-CSS</a></li>
+    <li><a href="https://sass-lang.com/">SCSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+
+  <ul>
+    <li><a href="https://nodejs.org/">Nodejs</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Clean and Simple Static Webpages]**
+- **[Only Raw HTML-CSS/SCSS-JS codes]**
+- **[Nodejs for development tools]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://mahabubx7.github.io/capstone-project-1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Usage
+
+To run the project, open the `index.html` in your browser.
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+Testing with Lint-checks:
+
+```sh
+  npm test
+```
+
+or, you can test linters for each tech
+```sh
+npm test:html
+```
+```sh
+npm test:styles
+```
+```sh
+npm test:js
+```
+
+### Deployment
+
+Deployed in `Github Pages`.
+[Live preview](https://mahbaubx7.github.io/capstone-project-1)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mahabub**
+
+- GitHub: [@mahabubx7](https://github.com/mahabubx7)
+- Twitter: [@mahabub__7](https://twitter.com/mahabub__7)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/mahabubx7)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give a start to this repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the `microverse` team for this nice ui (behance-shot).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
```

Comment:
- [ ] The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Kindly add it to the README of your project.

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/225587299-8d9b2dcf-711b-40c0-bb74-d445e6f5ebf6.png)